source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -849,7 +863,12 @@ See the `Load Balancer Specification <../load-balancers/load-balancers.rst#event
        * The service id for which the pool was cleared for in load balancing mode.
        * See load balancer specification for more information about this field.
        */
-      serviceId: Optional<ObjectId>
+      serviceId: Optional<ObjectId>;
+
+      /**
+       * A flag whether the pool forced closing "in use" connections as part of the clear.
+      */
+      closeInUseConnections: Optional<Boolean>;
```

Comment:
We should augment the tests to verify that this flag matches what we expect it to be.

Reply:
See my other comment about this: we should update the existing tests to assert on this flag rather than adding a new one.

Reply:
The current test and implementation designs don't allow doing it

Reply:
yeah, If I change a place where we trigger cleared pool event, then it's possible to simplify it, done